example/src/index.js

```diff
@@ -1,7 +1,7 @@
 import './index.css'
 
 import React from 'react'
-import ReactDOM from 'react-dom'
 import App from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+const root = createRoot(document.getElementById('root'))
```

Comment:
I suppose this is actually where I meant to ask the question about backwards compatibility.